AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -1,6 +1,3 @@
-# Current version of SwiftFormat used at Airbnb: 
-# https://github.com/calda/SwiftFormat/releases/tag/0.49.11-beta-2
```

Comment:
Since the SwiftFormat version has to be specified in `Package.swift`, I figure we can remove it from this file so we have a single source of truth